src/java.base/share/classes/com/sun/crypto/provider/DHPrivateKey.java

```diff
@@ -49,28 +49,28 @@ final class DHPrivateKey implements PrivateKey,
         javax.crypto.interfaces.DHPrivateKey, Serializable {
 
     @java.io.Serial
-    static final long serialVersionUID = 7565477590005668886L;
+    private static final long serialVersionUID = 7565477590005668886L;
 
     // only supported version of PKCS#8 PrivateKeyInfo
     private static final BigInteger PKCS8_VERSION = BigInteger.ZERO;
 
     // the private key
-    private BigInteger x;
+    private final BigInteger x;
 
     // the key bytes, without the algorithm information
-    private byte[] key;
+    private final byte[] key;
```

Comment:
I have an impending change which needs to re-assign key outside of constructor, thus can you leave this field as is and not marking it final?